diff --git a/node_modules/viem/_cjs/chains/definitions/gnosis.js b/node_modules/viem/_cjs/chains/definitions/gnosis.js
index 02550d2..f2daf0e 100644
--- a/node_modules/viem/_cjs/chains/definitions/gnosis.js
+++ b/node_modules/viem/_cjs/chains/definitions/gnosis.js
@@ -19,9 +19,9 @@ exports.gnosis = (0, defineChain_js_1.defineChain)({
     },
     blockExplorers: {
         default: {
-            name: 'Gnosisscan',
-            url: 'https://gnosisscan.io',
-            apiUrl: 'https://api.gnosisscan.io/api',
+            name: 'Gnosis Blockscout',
+            url: 'https://gnosis.blockscout.com',
+            apiUrl: 'https://gnosis.blockscout.com/api',
         },
     },
     contracts: {
diff --git a/node_modules/viem/_esm/chains/definitions/gnosis.js b/node_modules/viem/_esm/chains/definitions/gnosis.js
index c25458a..af1474b 100644
--- a/node_modules/viem/_esm/chains/definitions/gnosis.js
+++ b/node_modules/viem/_esm/chains/definitions/gnosis.js
@@ -16,9 +16,9 @@ export const gnosis = /*#__PURE__*/ defineChain({
     },
     blockExplorers: {
         default: {
-            name: 'Gnosisscan',
-            url: 'https://gnosisscan.io',
-            apiUrl: 'https://api.gnosisscan.io/api',
+            name: 'Gnosis Blockscout',
+            url: 'https://gnosis.blockscout.com',
+            apiUrl: 'https://gnosis.blockscout.com/api',
         },
     },
     contracts: {
diff --git a/node_modules/viem/chains/definitions/gnosis.ts b/node_modules/viem/chains/definitions/gnosis.ts
index dc98ffb..2f0f7d6 100644
--- a/node_modules/viem/chains/definitions/gnosis.ts
+++ b/node_modules/viem/chains/definitions/gnosis.ts
@@ -17,9 +17,9 @@ export const gnosis = /*#__PURE__*/ defineChain({
   },
   blockExplorers: {
     default: {
-      name: 'Gnosisscan',
-      url: 'https://gnosisscan.io',
-      apiUrl: 'https://api.gnosisscan.io/api',
+      name: 'Gnosis Blockscout',
+      url: 'https://gnosis.blockscout.com',
+      apiUrl: 'https://gnosis.blockscout.com/api',
     },
   },
   contracts: {
